tests/src/test/java/alluxio/server/ft/journal/ufs/UfsJournalIntegrationTest.java

```diff
@@ -104,10 +104,15 @@ public void addBlock() throws Exception {
     }
     os.close();
     URIStatus status = mFileSystem.getStatus(uri);
-    mLocalAlluxioCluster.stopFS();
+    stopFS();
     addBlockTestUtil(status);
   }
 
+  private void stopFS() throws Exception {
+    mLocalAlluxioCluster.stopFS();
+    Thread.sleep(500);
```

Comment:
Is there a reason why we have to sleep? Is there something else we could check on instead of sleeping?

Reply:
See my response to the other one..